storage/storage.go

```diff
@@ -73,9 +88,24 @@ type Store interface {
 	GetAllStats(ctx context.Context, queues []string) (*Stats, error)
 
 	// Heartbeat
-	SendHeartbeat(ctx context.Context, heartbeat *Heartbeat) error
-	RemoveHeartbeat(ctx context.Context, heartbeat *Heartbeat) error
```

Comment:
I like the symmetry of this interface, any reason to not keep the interface as-is?

Reply:
While the symmetry is nice, I think being specific about what is required is important. In my opinion it's unclear which properties of the Heartbeat struct are required for the remove to succeed, while the identifier makes it clear. This asymmetry is also common in other APIs (ie: REST has a payload for the POST, but usually just the identifier for the DELETE.)